src/stores/pacbioPoolCreate.js

```diff
@@ -601,7 +601,7 @@ export const usePacbioPoolCreateStore = defineStore('pacbioPoolCreate', {
         // after the request for all plates, as otherwise the partial record will over-write
```

Comment:
Can we remove the comment block above this now that the correct records are loaded?

Reply:
Done 👍